### DISCUSSION TITLE
Remplacer ESLint+Prettier par Biome

### DISCUSSION THREAD

**Discussion by ggrossetie** [EcrituresNumeriques/stylo#1160] [Ideas]
Semble être une bonne alternative à ESLint+Prettier avec une bonne compatibilité (97% compatible avec Prettier). Permet de lint/format du JavaScript, JSX, CSS et GraphQL.

https://biomejs.dev/

**Comment by ggrossetie** [comment]
Fait dans https://github.com/EcrituresNumeriques/stylo/pull/2023

**Comment by thom4parisot** [comment]
On a gagné quoi à cette migration ?

**Reply by ggrossetie** [reply]
Moins de dépendances à mettre à jour maintenir et un seul outil pour lint + format. On a supprimé 11 dépendances (ESLint + Prettier, directs) et 281 paquets si on inclut les dépendances transitives.

Il y a aussi des règles a11y inclus par défaut dans Biome qu'on pourra activer. Pour le moment j'ai désactivé un certain nombre de règles pour ne pas faire trop de changements dans le code.

**Reply by thom4parisot** [reply]
Super, merci 👍🏻

**Reply by thom4parisot** [reply]
D'ailleurs, à ce titre, pour faire un outil en moins — en se passant de Biome 😆 — ça serait de passer à [Vite+](https://viteplus.dev/)
